backend/canvas_app_explorer/tests.py

```diff
@@ -1,3 +1,16 @@
 from django.test import TestCase
+from django.conf import settings
+
+from .canvas_lti_manager import CanvasLtiManager
+from backend.canvas_app_explorer.data_class import ExternalTool
 
 # Create your tests here.
+class TestCanvasLtiManager(TestCase):
+    def test_get_ex_tools(self):
+        API_URL = settings.TEST_API_URL
+        API_KEY = settings.TEST_API_KEY
+        COURSE_ID = settings.TEST_COURSE_ID
+        manager = CanvasLtiManager(api_url=API_URL, api_key=API_KEY, course_id=COURSE_ID)
```

Comment:
You could just use `settings.API_URL`, etc. in-line when instantiating `CanvasLtiManager`.

Reply:
Yeah, these local variables aren't necessary anymore and probably should also be lower cased if they used. Just use the settings directly.